experimental/auth-logic/test_data/endorsement_wrapper_expected.auth_logic

```diff
@@ -0,0 +1,12 @@
+"oak_functions_loader:0f2189703c57845e09d8ab89164a4041c0af0a62::EndorsementFile" says {
+    // The endorsement policy claims a binary has an expected hash if... 
+    "oak_functions_loader:0f2189703c57845e09d8ab89164a4041c0af0a62::Binary" has_expected_hash_from("sha256:15dc16c42a4ac9ed77f337a4a3065a63e444c29c18c8cf69d6a6b4ae678dca5c", "oak_functions_loader:0f2189703c57845e09d8ab89164a4041c0af0a62::EndorsementFile") :-
+        // ... the real time is less than the given expiry time and greater
+        // than the release time stated in the endorsement file.
+        RealTimeNsecIs(current_time), current_time >= 1643710850, current_time < 1646130050.
+
+    // The endorsement policy trusts a time measurement policy called
+    // "UnixEpoch" to get the current time. (This just uses the local clock)
+    "UnixEpochTime" canSay RealTimeNsecIs(any_time).
+
+}
```

Comment:
This can't change without changing the way principals wrap the inner statements (we'd have to make it always add a newline at the end). I also can't seem to get these tests to pass either by adding `\n` or `\r\n`...

Reply:
Let's keep it as it then.